src/main/java/core/basesyntax/service/CsvFileWriter.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.Map;
+
+public class CsvFileWriter {
+    private static final String HEADER = "fruit,quantity" + System.lineSeparator();
+
+    public void write(String filepath, Map<String, Integer> storage) {
+        try {
+            Files.writeString(Paths.get(filepath), HEADER);
+            for (Map.Entry<String, Integer> entry : storage.entrySet()) {
+                Files.writeString(Paths.get(filepath),
+                        entry.getKey() + "," + entry.getValue() + System.lineSeparator(),
+                        StandardOpenOption.APPEND);
+            }
+        } catch (IOException e) {
+            System.out.println("Problems with writing to file");
```

Comment:
```suggestion
            System.out.println("Problems with writing to file", e);
```
add `e` as param. It will allow you to understand the problem you receive.

Reply:
done, thank you